.github/workflows/intermittent-test-check.yml

```diff
@@ -137,7 +137,8 @@ jobs:
           CHECK: ${{ needs.prepare-job.outputs.test_type }}
       - name: Summary of failures
         run: hadoop-ozone/dev-support/checks/_summary.sh target/${{ needs.prepare-job.outputs.test_type }}/summary.txt
-        if: always()
+        if: ${{ !cancelled() }}
+        continue-on-error: ${{ github.event.inputs.fail-fast != 'true' }}
```

Comment:
Can we fail the split step too instead of passing it in case of fail-fast = false. Just for better optics, If I look at the fail-fast = false run in the PR description, it looks like the Count Failure step failed but actually it failed the tests.

Reply:
If we fail the test split, the dependent "count" step will not be run:

https://github.com/adoroszlai/hadoop-ozone/actions/runs/6577494438

Reply:
Hmm, maybe we can add `if: ${{ always() }}` to achieve that.

Reply:
Right, In that case I guess even in the fail-fast = true run , the count step will get execute.

Reply:
Yes, that works:
https://github.com/adoroszlai/hadoop-ozone/actions/runs/6591518996